index.bs

```diff
@@ -2096,103 +2138,66 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
-                        1. Execute a [=client platform=]-specific procedure to determine which, if any, [=public key credentials=] described by
-                            <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> are [=bound credential|bound=] to this
-                            |authenticator|, by matching with |rpId|,
-                            <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}.{{PublicKeyCredentialDescriptor/id}}</code>,
-                            and
-                            <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}.{{PublicKeyCredentialDescriptor/type}}</code>.
-                            Set |allowCredentialDescriptorList| to this filtered list.
+        :   If an |authenticator| becomes available on this [=client device=],
+        ::  Note:  This includes the case where an |authenticator| was available upon |lifetimeTimer| initiation.
 
-                        1. If |allowCredentialDescriptorList| [=list/is empty=], [=continue=].
+            1. If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is {{CredentialMediationRequirement/conditional}}
+                and the |authenticator| supports the [=silentCredentialDiscovery=] operation:
 
-                        1. Let |distinctTransports| be a new [=ordered set=].
+                1. Let |collectedDiscoveredCredentialMetadata| be the result of invoking the [=silentCredentialDiscovery=] operation on |authenticator| with |rpId| as parameter.
 
-                        1. If |allowCredentialDescriptorList| has exactly one value, set
-                            <code>|savedCredentialIds|[|authenticator|]</code> to <code>|allowCredentialDescriptorList|[0].id</code>'s
-                            value (see [here](#authenticatorGetAssertion-return-values) in [[#sctn-op-get-assertion]] for more information).
+                1. [=list/For each=] |credentialMetadata| of |collectedDiscoveredCredentialMetadata|:
 
-                        1. [=list/For each=] credential descriptor |C| in |allowCredentialDescriptorList|,
-                            [=set/append=] each value, if any, of <code>|C|.{{transports}}</code> to |distinctTransports|.
+                    1. [=map/Set=] |silentlyDiscoveredCredentials|[|credentialMetadata|] to |authenticator|.
 
-                            Note: This will aggregate only distinct values of {{transports}} (for this [=authenticator=]) in
-                                |distinctTransports| due to the properties of [=ordered sets=].
+                    Note: A request will be issued to this authenticator upon user selection of a credential.
```

Comment:
Per our discussion, suggesting we expand this note and link to the switch case where the request is issued due to "...particular user context" -- I could not come up with a reasonable way to condense **"input** form control with an autocomplete attribute whose value contains a "webauthn" autofill detail token")

( I also indented it another level):
```suggestion
                        Note: A request will be issued to this authenticator upon user selection of a credential via
                        interaction with a particular UI context
                        (see [here](#GetAssn-ConditionalMediation-Interact-FormControl) for details).
```